scheduler/src/cook/kubernetes/api.clj

```diff
@@ -505,6 +505,7 @@
        (filter first) ; Keep those with non-nil node names.
        (pc/map-vals (fn [pods]
                       (->> pods
+                           (remove #(some-> % .getMetadata .getName synthetic-pod?))
```

Comment:
Let's put this behind a feature flag so that we can toggle it on/off via configuration.